lib/charms/mysql/v0/mysql.py

```diff
@@ -1778,6 +1778,24 @@ def _get_host_ip(host: str) -> str:
 
         return ",".join(rw_endpoints), ",".join(ro_endpoints), ",".join(no_endpoints)
 
+    def execute_remove_instance(self, connect_instance: Optional[str] = None) -> None:
+        """Execute the remove_instance() script with mysqlsh.
+
+        Args:
+            connect_instance: (optional) The instance from where to run the remove_instance()
+        """
+        remove_instance_options = {
+            "password": self.cluster_admin_password,
+            "force": "true",
```

Comment:
Q: do we need to force it by default?

Reply:
both of our use cases need force, but added a kwarg for `force=False` (tested against https://github.com/canonical/mysql-k8s-operator/actions/runs/10048296233)